rca/static_src/javascript/programmes/components/ProgrammeTeaser.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { programmePageShape } from '../programmes.types';
+
+/**
+ * A programme’s teaser info, to be displayed as part of search results.
+ */
+const ProgrammeTeaser = ({ programme }) => {
```

Comment:
This is a placeholder just for the form to be tested meaningfully – not worth reviewing now. Same for the corresponding styles.